dockers/docker-orchagent/ipinip.json.j2

```diff
@@ -44,12 +46,14 @@
 {% endfor %}
 {% for (name, prefix) in VLAN_INTERFACE|pfx_filter %}
     {%- if prefix | ipv4 %}
-        {%- set ipv4_addresses = ipv4_addresses.append(prefix) %}
+        {%- set ipv4_vlan_addresses = ipv4_vlan_addresses.append(prefix) %}
```

Comment:
ipv4_vlan_addresses.append will return None here

Reply:
Yes, this format is a little misleading here.
The `set` here actually defines a new `ipv4_vlan_addresses` in the inner block with value `None`, which is different from the outer `ipv4_vlan_addresses` defined in LINE#6.
But it works, as only the global level `ipv4_vlan_addesses` defined in LINE#6 is avaliable.

Reply:
I will keep using this syntax as all the templates are using this to append to list.